configs/sites/tier1/noaa-aws/README.md

```diff
@@ -16,3 +34,47 @@ spack concretize 2>&1 | tee log.concretize
 spack install --verbose 2>&1 | tee log.install
 spack module lmod refresh -y
 spack stack setup-meta-modules
+
+## Steps to install GSI addon
+
+sudo su -
+chmod 777 /contrib
+
+cd /contrib/spack-stack-rocky8/spack-stack-1.8.0
+. setup.sh
+spack stack create env --name gsi-intel-2021.10.0 --template gsi-addon-dev --site noaa-aws --upstream /contrib/spack-stack-rocky8/spack-stack-1.8.0/envs/ue-intel-2021.10.0/install --compiler intel
+cd envs/gsi-intel-2021.10.0
+spack concretize 2>&1 | tee log.concretize
+spack install --verbose 2>&1 | tee log.install
+spack module lmod refresh --upstream-modules
+spack stack setup-meta-modules
+
+## Steps to install external applications (MySQL, ecFlow)
+
+ MySQL:
+mkdir -p /contrib/spack-stack-rocky8/mysql-8.0.31/src
+cd /contrib/spack-stack-rocky8/mysql-8.0.31/src
+wget https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.31-linux-glibc2.17-x86_64-minimal.tar.xz
+tar -xvf mysql-8.0.31-linux-glibc2.17-x86_64-minimal.tar.xz
+mv mysql-8.0.31-linux-glibc2.17-x86_64-minimal/* ..
+rmdir mysql-8.0.31-linux-glibc2.17-x86_64-minimal
+
+ ecFlow:
```

Comment:
Why ecflow? I thought we had discussed that we don't need this anymore. But ok, let's leave it in the readme, it won't get used as long as the site config has it removed from the external packages and the qt5 external package added instead (if it isn't there already).